packages/webpack-plugin/test/e2e/stc-config-resolver.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { StylableProjectRunner } from '@stylable/e2e-test-kit';
+import { expect } from 'chai';
+import { dirname } from 'path';
+
+const project = 'stylable-config-resolver';
+const projectDir = dirname(
+    require.resolve(`@stylable/webpack-plugin/test/e2e/projects/${project}/package.json`)
+);
+
+describe(`(${project})`, () => {
+    const projectRunner = StylableProjectRunner.mochaSetup(
+        {
+            projectDir,
+            launchOptions: {
+                // headless: false
+            },
+        },
+        before,
+        afterEach,
+        after
+    );
+
+    it('config is resolved and css is working', async () => {
+        const { page } = await projectRunner.openInBrowser();
+        const { backgroundColor, color } = await page.evaluate(() => {
+            const style = getComputedStyle(document.body);
+            return {
+                color: style.color,
+                backgroundColor: style.backgroundColor,
+            };
+        });
+
+        expect(backgroundColor).to.eql('rgb(0, 128, 0)');
```

Comment:
would be nice to have expect labels on fail: `green from webpack alias` and `blue from typescript paths`

Reply:
added